weaviate/collection/collection.py

```diff
@@ -234,7 +235,9 @@ def near_object_options(
     def __result_to_object(self, obj: GrpcResult) -> _Object:
         data = obj.result
         for key in data.keys():
-            if isinstance(value := data[key], List):
+            entry = data[key]
+            if isinstance(entry, list):
+                value: List = entry
```

Comment:
This was a mypy warning? I'm surprised that it doesn't like the walrus operator, I think that it's a wonderfully expressive part of the language.

Reply:
yes, it didn't get the type of value before